CHANGELOG

```diff
@@ -2,6 +2,18 @@ libseccomp-golang: Releases
 ===============================================================================
 https://github.com/seccomp/libseccomp-golang
 
+* Version 0.11.0 - March 19, 2024
+- Add missing architectures (LOONGARCH64, M68K, SH, SHEB)
```

Comment:
nitpick - do we want to say `add new architectures` rather than `add missing architectures` :)

Reply:
makes sense, thanks!